src/towncrier/newsfragments/501.feature

```diff
@@ -0,0 +1 @@
+Don't raise an exception with ``towncrier check`` on missing configuration.
```

Comment:
very minor comment.

The release notes are for end users.

The current message sounds like a commit message, and less like a release note. maybe something like this.

And is this about about `towncrier check` or `towncrier` CLI in general?

```suggestion
Calling ``towncrier check`` without an existing configuration, will just show only an error message. In previous versions, a traceback was also generated.
```

Reply:
> In previous versions, a traceback was also generated.

Should the last version containing this and type of traceback (`ConfigError`) also be specified?